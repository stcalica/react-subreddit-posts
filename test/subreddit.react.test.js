import React from 'react';
import SubredditPosts from '../src/index';
import { shallow } from 'enzyme';
import componentDidMount from '../src/index';

describe('Subreddit Component Works', () => {

  test('That Subreddit Posts Loads', () => {
    const wrapper = shallow(
      <SubredditPosts
        subreddit="aww"
        display="gallery" //list, gallery, tile
        placeholder="http://www.codeisdead.com/img/profile/Kyle.jpg" // image if no image or preview or media
        width="250px" //of each tile or gallery post
        height="300px" //of each tile or gallery post
        //listings are always full page
      />);

  });

});
