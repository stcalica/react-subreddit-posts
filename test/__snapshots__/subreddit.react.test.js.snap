// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<ul
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
      "flexWrap": "wrap",
      "justifyContent": "flex-start",
      "listStyle": "none",
      "width": "100%",
    }
  }
/>
`;

exports[`Subreddit Component Works That Subreddit Posts Loads 1`] = `
<ul
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
      "flexWrap": "wrap",
      "justifyContent": "flex-start",
      "listStyle": "none",
      "width": "100%",
    }
  }
/>
`;

exports[`That Subreddit Posts Loads 1`] = `
<ul
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
      "flexWrap": "wrap",
      "justifyContent": "flex-start",
      "listStyle": "none",
      "width": "100%",
    }
  }
/>
`;
